src/Server.php

```diff
@@ -184,16 +190,16 @@ public function getBaseUrl()
 
     /**
      * Set cache file system.
-     * @param FilesystemInterface $cache Cache file system.
+     * @param FilesystemInterface|FilesystemOperator $cache Cache file system.
      */
-    public function setCache(FilesystemInterface $cache)
+    public function setCache($cache)
```

Comment:
While the chances of some extending the `Server` class are quite low, changing the signature of a public method is a BC break as per semver.